packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1658,6 +1644,96 @@ internal class DdSdkTest {
             }
     }
 
+    @Test
+    fun `𝕄 initialize native SDK 𝕎 initialize() {small batch size}`(
```

Comment:
minor: since the size of `DdSdkTest` file is growing, maybe it is worth to convert some similar tests to `ParametrizedTest`? So that there is once test with just different inputs.

Reply:
I was looking for something like this! Thanks a lot I will update some of the tests that will benefit from it :)

Reply:
We are using them in the Android SDK codebase, you can check there.